test/commons/builder/executive_test.rb

```diff
@@ -51,4 +51,18 @@ def test_list_with_missing_position
     executives = Executive.list(config)
     assert_equal [], executives
   end
+
+  def test_list_with_two_positions_for_one_executive
+    # If an executive is listed in the results more than once, with multiple positions, group those positions into a
+    # single executive entry. Also ensure that the entries are sorted by executive and position item IDs.
+    stub_request(:post, 'https://query.wikidata.org/sparql')
+      .to_return(body: open('test/fixtures/executive-index-two-positions.srj', 'r')).then
+      .to_return(body: '[]')
```

Comment:
I think the second `.then.to_return` is unnecessary here (the test pass without it).